frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -14,7 +14,7 @@ const BucketsRowStatus: React.FC<BucketsRowStatusProps> = React.memo(({ status,
       <span className="co-dashboard-text--small nb-buckets-card__row-subtitle">Unavailable</span>
     ) : Number(status) > 0 ? (
       <React.Fragment>
-        <a href={link} style={{ textDecoration: 'none' }}>
+        <a href={link} style={{ textDecoration: 'none' }} target="blank">
```

Comment:
Make this a external link. make sure all the noobaa dashboard links are external.

Reply:
its external !

Reply:
https://github.com/openshift/console/blob/master/frontend/public/components/nav/items.tsx#L22..L24

Reply:
how would I wrap this in`ExternalLink`
```
<RedExclamationCircleIcon className="co-dashboard-icon nb-bucket-card__status-icon" />
 <span className="nb-buckets-card__row-status-item-text">{status}</span>
```